### PR TITLE
Add votes (and corresponding test!)

### PR DESCRIPTION
**Pull request by suzicurran**

### CODE REVIEW

File: app/controllers/votes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class VotesController < ApplicationController
+  before_action :set_review
+  before_action :set_monster
+
+  def handle_vote
+    matching_votes = Vote.where(user_id: current_user.id, review_id: @review.id)
+    vote_value = params[:value]
+    if matching_votes.empty?
+      @vote = Vote.new(user_id: current_user.id, review_id: @review.id, value: vote_value)
+      if @vote.save
```
[the remark this answers is not in the document]
Comment (suzicurran) [reply app/controllers/votes_controller.rb:10]: Doesn't make sense in this context where we're only doing assignment in one of the conditions.

[the remark this answers is not in the document]
Comment (suzicurran) [reply app/controllers/votes_controller.rb:10]: See above.

File: app/controllers/votes_controller.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+class VotesController < ApplicationController
+  before_action :set_review
+  before_action :set_monster
```
Comment (justinhuynh) [inline app/controllers/votes_controller.rb:3]: Just FYI you can do `before_action :set_review, :set_monster` all at once

File: app/views/monsters/show.html.erb [hunk 3]
```diff
@@ -30,6 +30,14 @@
   <ul>
     <% @reviews.each do |review| %>
     <li><%= render partial: 'review', locals: {review: review} %></li>
+    <div>
+      <%= render partial: 'upvote', locals: {review: review} %> <%= render partial: 'downvote', locals: {review: review} %>
+    </div>
+    <div>
+      <% upvotes = Vote.where(review_id: review.id, value: "+") %>
```
Comment (justinhuynh) [inline app/views/monsters/show.html.erb:37]: I would try to do this in the controller. You can assign the instance variable `@upvotes` and then use it directly on line 39. Same for `downvotes`

**Review by justinhuynh — APPROVED** [review]
Looks good. Minor suggested changes

**Pull request merged**